### DISCUSSION TITLE
Categories without content

### DISCUSSION THREAD

**Discussion by jvanvolken** [photoprism/photoprism#1386] [Ideas]
Howdy, PhotoPrism team and community!

Categories without content should not be listed or at least grayed out. For example, I have no videos from this Samsung device, yet I can select it from the drop-down. 
![image](https://user-images.githubusercontent.com/43481443/122791420-1249e400-d26e-11eb-95d6-5180717b61a7.png)

This gets tedious when you have a large selection of cameras and most do not have videos.
![image](https://user-images.githubusercontent.com/43481443/122791590-373e5700-d26e-11eb-9a63-2d173686c51a.png)

On top of this, when you select a filter, it should remove or gray out other filters that will result in a zero content result. For example, I would like to know which cameras were used at Saint Lucia, instead of having to select each device in order to find which one has pictures. 
![image](https://user-images.githubusercontent.com/43481443/122793283-dd3e9100-d26f-11eb-8cd5-9460358b9c50.png)

**Comment by lastzero** [comment]
That would require a search in the backend database after each change / click, which adds complexity and reduces UI performance. So we've chosen no to do this for now until we find a solution with low overhead.

**Reply by jvanvolken** [reply]
Thanks for the response! 

That certainly makes sense. I'm sure it would slow down the response slightly; however, the interface returns the "Couldn't find anything" message nearly instantaneously - certainly faster than looking by hand. Maybe an option in the settings to enable this would work.
